src/main/java/com/gizasystems/automationexercise/pages/CategoriesBar.java

```diff
@@ -0,0 +1,36 @@
+package com.gizasystems.automationexercise.pages;
+
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+
+public class CategoriesBar {
+    // Variables
+    private SHAFT.GUI.WebDriver driver;
+
+    // Locators
+    private final By category_div = By.xpath("//div[@class='panel-group category-products']");
+    public By getCategoryLinkLocator(String categoryType){
+        return  By.xpath("//a[@href='#"+categoryType+"']");
+    }
+    public By getSubCategoryLinkLocator(String categoryType,String subCategoryType ){
+        return  By.xpath("//div[@id='"+ categoryType +"']//a[contains(text(),'"+ subCategoryType +"')]");
+    }
+
+    // Constructor
+    public CategoriesBar(SHAFT.GUI.WebDriver driver) {
+        this.driver = driver;
+    }
+
+    //////////////////// Actions \\\\\\\\\\\\\\\\\\\\
+    @Step("click on Category Link")
+    public CategoriesBar clickOnCategoryLink(String categoryType) {
+        driver.element().click(getCategoryLinkLocator(categoryType));
+        return this;
+    }
+    @Step("click on SubCategory Link")
+    public CategoriesBar clickOnSubCategoryLink(String categoryType,String subCategoryType) {
+        driver.element().click(getSubCategoryLinkLocator(categoryType,subCategoryType));
+        return this;
+    }
+}
```

Comment:
I believe the category section is only found in the Home page, no need to create a whole page for it. (this can be done with the single page systems only)

Reply:
We can resolve this comment as it found in the Products page.